icu4c/source/common/uprops.cpp

```diff
@@ -590,7 +590,11 @@ static int32_t scriptGetMaxValue(const IntProperty &/*prop*/, UProperty /*which*
 
 /*
  * Map some of the Grapheme Cluster Break values to Hangul Syllable Types.
- * Hangul_Syllable_Type is fully redundant with a subset of Grapheme_Cluster_Break.
+ * Hangul_Syllable_Type is redundant with a subset of Grapheme_Cluster_Break.
```

Comment:
I don't think just deleting "fully" does the trick here.  I think you either need to say something like "Hangul_Syllable_Type is mostly redundant with a set of Grapheme_Cluster_Break, but starting in Unicode 16, there is an exception..." or "HST used to be fully redundant with a set of GCB, but starting in Unicode 16, this is no longer fully true..."